app/views/patients/index.csv.erb

```diff
@@ -0,0 +1,8 @@
+<%=
```

Comment:
The one upside to using the `CSV.generate` is we don't have to handle the special case of escaping commas by quoting any affected values.  This is the only special case I can think of, but I like using purpose-built libraries because we don't have to think about the edge cases.